src/cache/redis-cache.js

```diff
@@ -1,5 +1,6 @@
 const redis = require('redis')
 const { promisify } = require('util')
+const storeAll = require('../utils/store-odp-data')
```

Comment:
We shouldn't initialize here. We should handle this logic inside the controller.

Reply:
Moved storage operations to when the options are called